src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,55 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Todo[];
+  handleShow: (todo: Todo) => void;
+  selectedTodoId?: number;
+}
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-      <tr data-cy="todo" className="has-background-info-light">
-        <td className="is-vcentered">2</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">quis ut nam facilis et officia qui</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye-slash" />
-            </span>
-          </button>
-        </td>
-      </tr>
+export const TodoList: React.FC<Props> = ({
+  todos,
+  handleShow,
+  selectedTodoId,
+}) => {
+  return (
+    <table className="table is-fullwidth is-striped">
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Title</th>
+          <th>Status</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        {todos.map(todo => (
+          <tr key={todo.id} data-cy="todo">
+            <td>{todo.id}</td>
+            <td>{todo.title}</td>
+            <td>
+              {todo.completed ? (
+                <span data-cy="iconCompleted">✔️</span>
+              ) : (
+                <span data-cy="iconPlanned">✖️</span>
+              )}
+            </td>
+            <td>
+              <button data-cy="selectButton" onClick={() => handleShow(todo)}>
```

Comment:
The name of your event handler `handleShow` is not clearly describing its purpose. According to the best practices, it's better to name event handlers as `onEventNameForElement`.